turbinia/client.py

```diff
@@ -138,13 +138,18 @@ def register_job_timeouts(dependencies):
     dependencies(dict): dependencies to grab timeout value from.
   """
   log.info('Registering job timeouts.')
+  timeout_default = 3600
 
   job_names = list(job_manager.JobsManager.GetJobNames())
   # Iterate through list of jobs
   for job, values in dependencies.items():
     if job not in job_names:
       continue
     timeout = values.get('timeout')
+    if type(timeout) != int:
+      log.warning(
+          'No timeout found for job: {0:s}. Setting default timeout of {1:d} seconds.'
+          .format(job, timeout_default))
```

Comment:
I think you still need to set the timeout value to the default :).

Reply:
Then in the register command below, you probably want to use the newly set `timeout` instead of getting it from the `values` dictionary again.

Reply:
Ah good find, updated.